include/GLFW/glfw3.h

```diff
@@ -2751,6 +2760,36 @@ GLFWAPI void glfwHideWindow(GLFWwindow* window);
  */
 GLFWAPI void glfwFocusWindow(GLFWwindow* window);
 
+/*! @brief Starts a resize operation with the specified window.
+ *
+ *  This function starts a resize operation on one of the borders of the
+ *  specified window.
```

Comment:
You should add a mention that this function must be called from a pointer event callback, otherwise it risks reacting to a different event.

Reply:
Or touch.